code/TagField.php

```diff
@@ -310,7 +310,10 @@ protected function getTags($term) {
 		$term = Convert::raw2sql($term);
 
 		$query = $dataClass::get()
-			->filter($titleField . ':PartialMatch:nocase', $term)
+			->filter(array(
```

Comment:
`TagField` is meant to be able to work with any `DataList`. The source tags depend on the source `DataList` that you provide the constructor/factory...